Gemfile

```diff
@@ -1,2 +1,9 @@
 source 'https://rubygems.org'
-gemspec
\ No newline at end of file
+gemspec
+
+# this is temporary for the feature/plugin-api-2_0 branch and is meant for travis testing
+gem "logstash-core", :path => "./logstash/logstash-core"
```

Comment:
curious, won't this hinder plugin authors? Like if they 'git clone' this repo and want to run the test suite, it's probably not going to find logstash-core in ./logstash/logstash-core